### DISCUSSION TITLE
Add to the Accrescent.app store

### DISCUSSION THREAD

**Discussion by JaByteOS** [anonaddy/anonaddy#663] [Feature Requests]
Have you considered or are you planning on putting your app on Accrescent (app store focused on security, privacy, and usability) 

Accrescent is a private and secure Android app store built with modern features in mind.

It was just endorsed by GrapheneOS (considered by many to be the most private and secure mobile operating). GrapheneOS also now has accrescent in their app store. 

Right now there is only 9 apps on it. So any new apps added get great installation opportunities from the privacy and security minded community. 

Anyway, I really like your app, and would love to see it on Accrescent because it's so easy to install and update apps compared with f-droid, plus much more secure! 

Thank you for taking the time to read this. I'm a security and privacy conscious individual, not associated with Accrescent or GrapheneOS (though I use them both).

Xoxo Olivia

**Comment by roke-julian-lockhart** [comment]
> https://github.com/anonaddy/anonaddy/discussions/663#discussion-6959314

@JaByteOS, have you read https://github.com/anonaddy/anonaddy/blob/bc626e80bfc3e21a8f9ae39fce987a33ad795e2a/README.md#is-there-an-android-app?
